runtime-library/CMakeLists.txt

```diff
@@ -37,12 +44,14 @@ endif()
 # development as it precludes important optimizations.
 option(BUILD_SHARED_LIBS "Build development shared runtime library" OFF)
 
-# When building a shared library, further customize:
-#   - Build with hidden visibility by default.
-#   - Unhide API exported functions.
-#   - Mark symbols for dllexport on Windows by default.
+# When building a shared library, further customize: - Build with hidden
```

Comment:
I think your editor was being a bit too helpful :-)

Reply:
It seems that cmake-format did this. I changed the way of writing so we could keep using cmake-format.